### ISSUE TITLE
Incorrect formatting of text output.

### ISSUE THREAD

**Issue by pehkawn** [jcrodriguez1989/chatgpt#47]
When calling `chatgpt()` from R, the text body's formatting is lost. Any line breaks, numbered list or code blocks in the text is gone. The text body includes symbols, such as /n, /n1 and ```, that I presume denotes the aforementioned, respectively. In a Jupyter notebook, I tried changing the output from HTML to Markdown. This will change any code to inline code. Line breaks and lists are still not displayed correctly. 

It would be helpful if the function had the ability to return ChatGPT's answer with the correct formatting.

**Comment by LuisLauM** [comment]
> When calling `chatgpt()` from R, the text body's formatting is lost. Any line breaks, numbered list or code blocks in the text is gone. The text body includes symbols, such as /n, /n1 and ```, that I presume denotes the aforementioned, respectively. In a Jupyter notebook, I tried changing the output from HTML to Markdown. This will change any code to inline code. Line breaks and lists are still not displayed correctly.
> 
> It would be helpful if the function had the ability to return ChatGPT's answer with the correct formatting.

It would be better if you could offer a reproducible example. In my case, I have no issues:
R 4.3.1
RStudio 2023.06.1 Build 524
Windows 10
chatgpt 0.2.3

![image](https://github.com/jcrodriguez1989/chatgpt/assets/8452334/599186b1-7f53-4e2a-abdc-a470c3bd2b52)

**Comment by pehkawn** [comment]
I realised after posting, that I didn´t wrap the function in `cat()`. Doing so did render a similar output to yours. This does not format the code snippets, however, but I found out that I can achieve this using the `IRdisplay::display_markdown()`. This will render a nicely formatted output in a Jupyter notebook.

Example:
![image](https://github.com/jcrodriguez1989/chatgpt/assets/44672121/cb37ba8a-4040-4567-b784-35a0beb6be38)

Still, it might be nice if the function had a second argument where output formatting is selected; for example, by adding a formatting argument (e.g. `ask_chatgpt(question, formatting = "plain"/markdown"/"html"/"latex") to the `ask_chatgpt()` function.
